File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SteakBraise.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food;

/* JADX INFO: loaded from: classes3.dex */
public class SteakBraise extends Food {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable, it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.food_steak_braise_name;
        this.idDescription = R.string.food_steak_braise_description;
        this.idImage = R.drawable.steak_braise;
        this.price = 24L;
        this.feedPower = 65;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SteakBraise.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food

class SteakBraise : Food() {
    override fun configureProperties() {
        idName = R.string.food_steak_braise_name
        idDescription = R.string.food_steak_braise_description
        idImage = R.drawable.steak_braise
        price = 24L
        feedPower = 65
    }
}
